sdk/src/client/secret/mnemonic.rs

```diff
@@ -147,6 +206,26 @@ impl MnemonicSecretManager {
         let seed = Seed::from_bytes(bytes.as_ref());
         Ok(Self(seed))
     }
+
+    /// Generate a random mnemonic to use for the secret manager.
+    pub fn generate_random() -> Result<Self, Error> {
```

Comment:
This might be nice for testing, but looks dangerous to have public, because one has no access to the mnemonic/seed, so when it's dropped all control over potential generated addresses is gone

Reply:
Isn't it already publicly available?

Reply:
I only see it added in this PR

Reply:
I moved it to the tests common module